test/config_SUITE.erl

```diff
@@ -1086,3 +1087,15 @@ config(_Config)  ->
     ?error_option(set_cfg_value([path_management, idle_timeout], invalid, ?PGW_PROXY_CONFIG)),
     ?error_option(set_cfg_value([path_management, down_timeout], invalid, ?PGW_PROXY_CONFIG)),
     ok.
+
+ergw_c_node_config() ->
+    [{doc, "Test for integrity of the configuration file ergw-c-node.config"}].
+ergw_c_node_config(_Config) ->
+    {ok, Cwd} = file:get_cwd(),
+    [Path | _] = re:split(Cwd, "_"),
```

Comment:
Add a comment to describe what this is doing.
Or change the code to
lists:takewhile( fun(X) -> X =/= "_build" end, filename:split(Path) )